### DISCUSSION TITLE
update rule

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#4017] [Q&A] [answered]
` @request.data.price_max >= price_max  `
 this update rule will not  applicable if  
`@request.auth.reference.Account_type = 'Association'    `
how to do that?

**Comment by ganigeorgiev** [comment, marked as answer]
If you are looking for a "either this or the other" type of rule, then a simple `||` (OR) expression should work:

```js
@request.auth.reference.Account_type = 'Association' ||
@request.data.price_max >= price_max
```

**Reply by apujayed** [reply]
update rule :
`@request.auth.reference.Account_type !== 'Association' ? @request.data.price_max >= price_max : null;`

This is not working. Can you help me achieve the similar output in PocketBase?
my requirement is if the Account_type !== 'Association'  only than the update rule will be applicable.

**Reply by ganigeorgiev** [reply]
This is exactly what my example is supposed to do.

If the `Account_type` is "Association" the second part of the rule will not be applied.

I have no context about your available collection fields and structure. If the example doesn't work, then double check your collection fields and the submitted request data.
